src/Lykke.Service.KycReports/Controllers/KycReportingController.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System;
+using System.Threading.Tasks;
+
+using Microsoft.AspNetCore.Mvc;
+using Lykke.Service.KycReports.Core.Domain.Reports;
+using System.Collections.Generic;
+using Lykke.Service.Kyc.Abstractions.Domain.Verification;
+
+namespace Lykke.Service.KycReports.Controllers
+{
+    //[Authorize]
+    [Route("api/[controller]")]
+    public class KycReportingController : Controller
+    {
+        private readonly IKycReportingService _kycReportingService;
+
+        public KycReportingController(IKycReportingService kycReportingService) {
+            _kycReportingService = kycReportingService;
+        }
+
+        [HttpGet]
+        [Route("stats/{dateFrom}/{dateTo}")]
+        public async Task<string> GetKycOfficerStatsJsonAsync(DateTime dateFrom, DateTime dateTo)
+        {
+            return await _kycReportingService.GetKycOfficerStatsJsonAsync(dateFrom, dateTo);
```

Comment:
From the called logic it seems that Date parts of DateTime parameters are expected. Consider pass dateFrom.Date and dateTo.Date to called method. Same goes to all other methods in this controller.

Reply:
changed